nlp/src/test/java/hivemall/nlp/tokenizer/SmartcnUDFTest.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package hivemall.nlp.tokenizer;
+
+import java.io.IOException;
+import java.util.List;
+
+import org.apache.hadoop.hive.ql.exec.UDFArgumentException;
+import org.apache.hadoop.hive.ql.metadata.HiveException;
+import org.apache.hadoop.hive.ql.udf.generic.GenericUDF;
+import org.apache.hadoop.hive.ql.udf.generic.GenericUDF.DeferredObject;
+import org.apache.hadoop.hive.serde2.objectinspector.ObjectInspector;
+import org.apache.hadoop.hive.serde2.objectinspector.ObjectInspectorFactory;
+import org.apache.hadoop.hive.serde2.objectinspector.primitive.PrimitiveObjectInspectorFactory;
+import org.apache.hadoop.io.Text;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class SmartcnUDFTest {
+
+	@Test
+	public void testOneArgment() throws UDFArgumentException, IOException {
+		GenericUDF udf = new SmartcnUDF();
+		ObjectInspector[] argOIs = new ObjectInspector[1];
+		// line
+		argOIs[0] = PrimitiveObjectInspectorFactory.javaStringObjectInspector;
+		udf.initialize(argOIs);
+		udf.close();
+	}
+
+	@Test
+	public void testTwoArgment() throws UDFArgumentException, IOException {
+		GenericUDF udf = new SmartcnUDF();
+		ObjectInspector[] argOIs = new ObjectInspector[2];
+		// line
+		argOIs[0] = PrimitiveObjectInspectorFactory.javaStringObjectInspector;
+		// stopWords
+		argOIs[1] = ObjectInspectorFactory
+				.getStandardConstantListObjectInspector(
+						PrimitiveObjectInspectorFactory.javaStringObjectInspector,
+						null);
+		udf.initialize(argOIs);
+		udf.close();
+	}
+
+	@Test
+	public void testEvalauteOneRow() throws IOException, HiveException {
```

Comment:
Typo `Evalaute` => `Evaluate`

I know the typo originally comes from `KuromojiUDFTest.java` :) Could you fix both typos?